_inc/client/unified-importer/author-mapping.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { withRouter } from 'react-router';
+import debugModule from 'debug';
+import apiFetch from '@wordpress/api-fetch';
+import { Button, CheckboxControl } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+// @TODO: is this the correct way to handle styles?
```

Comment:
For now, we might want to enqueue `wp-components` stylesheet to give us the baseline `@wordpress/component` styles, though I'm unsure if we'll want to continue to do this or not as we add custom styling.